.github/workflows/deb-builder.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: ['ubuntu:22.04', 'ubuntu:20.04', 'debian:stable', 'debian:testing', 'debian:unstable']
+        os: ['ubuntu:22.04', 'ubuntu:20.04', 'debian:stable', 'debian:testing', 'debian:unstable', 'debian:oldstable']
```

Comment:
```suggestion
        os: ['ubuntu:22.04', 'ubuntu:20.04', 'debian:oldstable', 'debian:stable', 'debian:testing', 'debian:unstable']
```

Reply:
this is just my OCD hitting, you can merge without the chronological order of Debian versions